app/models/form_submission_attempt.rb

```diff
@@ -25,14 +25,21 @@ class FormSubmissionAttempt < ApplicationRecord
 
     event :fail do
       after do
+        log_info = { form_submission_id:,
+                     benefits_intake_uuid: form_submission&.benefits_intake_uuid,
+                     form_type: form_submission&.form_type }
         if should_send_simple_forms_email
-          Rails.logger.info({
-                              message: 'Preparing to send Form Submission Attempt error email',
-                              form_submission_id:,
-                              benefits_intake_uuid: form_submission.benefits_intake_uuid,
-                              form_type: form_submission.form_type
-                            })
+          Rails.logger.info('Preparing to send Form Submission Attempt error email', log_info)
           simple_forms_enqueue_result_email(:error)
+        else
+          # Do not love hard coding this in here like this.
+          # Hoping to refactor this at somepoint to better support various emailing on arbitrary or inherited classes
+          should_send_form526_form4142_email
+          form526_submission_id = Form526Submission.find_by(saved_claim_id:).id
+          Rails.logger.info('Queuing Form526:Form4142 failure email to VaNotify',
+                            log_info.merge({ form526_submission_id: }))
+          jid = EVSS::DisabilityCompensationForm::Form4142DocumentUploadFailureEmail.perform_async form526_submission_id
```

Comment:
Just a thought but we have our failure emails scheduled to send at 9:00am ET so they don't go out at odd hours. Not a blocker by any means though! 🙂

Reply:
Interesting... we blast all ours out whenever, but I will take this topic to our PM/Research/UX people to see if they'd rather us do something like that. I think our thought was telling the user as soon as possible, is better than potentially up to 23hrs later, but, we can think on that.